Source/ACE/Managers/WorldManager.cs

```diff
@@ -10,14 +10,17 @@
 using ACE.Entity;
 using ACE.Network;
 
+using log4net;
+
 namespace ACE.Managers
 {
     public static class WorldManager
     {
-        // commented: unused.  uncomment if you'll use it.
-        // private static uint sessionTimeout = 150u; // max time between packets before the client disconnects
+        private static readonly ILog log = LogManager.GetLogger(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);
 
-        private static readonly List<Session> sessionStore = new List<Session>();
+        public const ushort ServerId = 0xB;

```

Comment:
It was already B, just hard coded buried somewhere.  It honestly doesn't matter what it is currently as we are single process.  If we ever move to a multi-process/multi-server, then this would need to be changed to allow for that.  I can add a comment to that effect.